README.md

```diff
@@ -100,6 +100,26 @@ $ docker volume rm MyVolume
 $ /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls
 ```
 
+## Upgrade Instructions
+
+User's data is not impacted during driver or plugin upgrade.
+vDVS managed plugin is taking care of the control path (management layer) and doesn't deal with data path. So during the upgrade process, only management related operations will be affected and data will be intact. Management operations like attach/detach/create/delete volume using vDVS driver may run into an error for the time being until the upgrade process is finished.
+
+Steps for safe upgrade:
+1. Uninstall VIB (vDVS driver)
+```esxcli software vib remove -n esx-vmdkops-service```
+2. Download and install the latest VIB from the vDVS release page
+```esxcli software vib install -v /tmp/<vib_name>.vib```
+3. Uninstall the managed plugin from the docker host
+```docker plugin rm -f <plugin name>```
+4. Install the latest release of vDVS managed plugin
+```docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest```
+
+We do not require user to stop containers to do plugin upgrade.
```

Comment:
Sure? if container is running and then plugin is removed and then container stops who detaches the plugin. Same for service.